content/docs/user-guide/gems/reference/physics/nvidia/recast-navigation.md

```diff
@@ -0,0 +1,19 @@
+---
+linkTitle: Recast Navigation
+title: Recast Navigation Gem
+description: The Recast Navigation Gem provides navigation mesh calculation and path finding functionality and visualizations in Open 3D Engine (O3DE) projects.
+toc: true
+---
+
+The Recast Navigation Gem provides support for building navigation meshes and calculating walkable paths within those navigation meshes. It uses open source library [Recast Navigation](https://github.com/recastnavigation/recastnavigation).
+
+An example of its use can be found in AutomatedTesting project in Navigation Sample level.
+
+This Gem includes the following features:
++ Select an area to build a navigation mesh from PhysX colliders
++ Create a navigation mesh
++ Configure navigation mesh parameters
++ Find paths between entities or positions within a navigation mesh
++ Visualize the navigation mesh
```

Comment:
question: is `+` the preferred character for bulleted lists? I think it's generally preferred to use `-` but I'm not 100% - I definitely recommend installing Markdown Lint for VSCode if you're using it as it highlights a lot of these types of conventions

Reply:
Awesome thanks!